### ISSUE TITLE
Planting grass on the farm (short one)

### ISSUE THREAD

**Issue by Rappeldrache** [StardewModders/mod-ideas#42]
Hello,

I would love to have a mod that makes it possible to plant grass (on the farm). The NORMAL gras (NOT the long one that gives you hay). So you can turn dirt whenever you want into gras. Your whole farm could get a nother "visual appearance" with this.
In combination with the mod "Plant on grass" (here: http://www.nexusmods.com/stardewvalley/mods/1026/? ) it would be incredible!
Should be "normal grass". If you don't have the "Plant on grass" mod you can't even plant on it. 

Thanks :-)

**Comment by pcsbx** [comment]
Couldn't this be done rather easily with a retextured/new flooring?

**Comment by Acerbicon** [comment]
@pcsbx No. Because it wouldn't behave as grass. Weeds and trees and wouldn't grow on it, nor would you be able to plant on it if you had the mod that let's you plant on grass. 

If all someone wanted was something that looked like grass, but didn't behave as it, then that could be done with flooring.

**Comment by Rappeldrache** [comment]
It's about to design the farm with "dirt" and "grass", with trees & bushes and whatever. Ingame, till you play. If I just change the dirt color, this is no problem ... But I mean "really grass". ;)

**Comment by KeepFireFlying** [comment]
This isn't really a duplicate since it's older, but I think Terraforming (#96) might address this. Is that what you're looking for?

**Comment by tiakall** [comment]
It's newer, actually, and emcompasses the same issue. Closing in favor of newer ticket.
